cmd/selfupdate/selfupdate_test.go

```diff
@@ -0,0 +1,146 @@
+package selfupdate
+
+import (
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"regexp"
+	"runtime"
+	"testing"
+
+	"github.com/rclone/rclone/fs"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestInstallOnLinux(t *testing.T) {
```

Comment:
These tests are downloading real files aren't they?

Which makes them more integration tests as if the download server is down the tests will fail.

We could run a test server to serve the files and have an Opt to configure the URLs?

Or we could only run these tests on the integration test server and skip them on the CI with fstest/testy.SkipUnreliable ?

Reply:
Testing against a dummy rclone.org doesn't look like a convincing weighty test to me.
In the same time, yeah, the test will fail if downloads.rclone.org or beta.rclone.org breaks down.
I'd constrain it also for the DDoS reason as every force-push on every PR on every rclone fork will send few more requests to rclone.org
I tried to tackle it with the `!testing.Short` check. I'll use `testy.SkipUnreliable` after your advice.